src/app/core/user/user-list/user-list.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, Input } from "@angular/core";
+import { User } from "../user";
+import { EntityMapperService } from "../../entity/entity-mapper.service";
+import { OnInitDynamicComponent } from "../../view/dynamic-components/on-init-dynamic-component.interface";
+import { LoggingService } from "../../logging/logging.service";
+
+@Component({
+  selector: "app-user-list",
+  template: `
+    {{ authorNames }}
+    <span *ngIf="_users.length > maxUserThreshold">
+      and {{ additionalUsers }} more
+    </span>
+  `,
+})
+export class UserListComponent implements OnInitDynamicComponent {
+  _users: User[] = [];
+
+  @Input() set entities(entities: (User | string)[]) {
+    if (this.inputType === "id") {
+      this.entityMapperService
+        .loadType<User>(User)
+        .then((users) => {
+          this._users = users.filter((u) =>
+            entities.find((e: string) => e === u.getId())
+          );
+        })
+        .catch((error) => {
+          this.loggingService.error(error);
+        });
+    } else {
+      this._users = entities as User[];
+    }
+  }
+  @Input() inputType: "id" | "entity" = "id";
```

Comment:
Same as before, I think the input type can be taken from the type of input to the `set entities` function to reduce the amount of variables